src/main.rs

```diff
@@ -115,6 +118,7 @@ where
 {
     let running = Arc::new(AtomicBool::new(true));
     let paused = Arc::new(AtomicBool::new(false));
+    let dns_shown = Arc::new(AtomicBool::new(opts.render_opts.show_dns));
```

Comment:
Why does this need to be an `Arc<AtomicBool>` - do you think we can make it a normal boolean and then pass it around without cloning or anything? Seems easier to me.

Reply:
I just thought that we could switch between `show_dns` modes via some key. But, yeah, i will replace it, or maybe should I add switching functionality to it?